### PR TITLE
recipes-multimedia: Update for NXP release 6.1.55-2.2.0

### PR DESCRIPTION
**Pull request by valijec**
Upgraded:
 - gstreamer1.0-plugins-bad:     1.22.0 -> 1.22.5
 - gstreamer1.0-plugins-base:    1.22.0 -> 1.22.5
 - gstreamer1.0-plugins-good:    1.22.0 -> 1.22.5
 - gstreamer1.0:                 1.22.0 -> 1.22.5
 - imx-codec:                    4.8.1 -> 4.8.2
 - imx-dsp:                      2.1.5 -> 2.1.6
 - imx-dsp-codec-ext:            2.1.5 -> 2.1.6
 - imx-parser:                             4.8.1 -> 4.8.2

Updated:
 - imx-alsa-plugins_git.bb
 - imx-gst1.0-plugin_git.bb
 - imx-vpuwrap_git.bb

New feature:
 - imx-opencl-converter_0.1.bb

### CODE REVIEW

**Comment by valijec** [comment]
Fixes issue [#1715](https://github.com/Freescale/meta-freescale/issues/1715)

File: recipes-multimedia/imx-parser/imx-parser_4.8.1.bb
Comment (thochstein) [inline recipes-multimedia/imx-parser/imx-parser_4.8.1.bb:1]: Version needs to be updated

**Review by thochstein — CHANGES_REQUESTED** [review]
There are some other changes in meta-imx that we might need, like `pipewire`

**Review by MrCry0 — APPROVED** [review]

File: recipes-multimedia/gstreamer/imx-gst1.0-plugin_git.bb [hunk 1]
```diff
@@ -22,7 +34,7 @@ RCONFLICTS:${PN} = "gst1.0-fsl-plugin"
 
 LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=fbc093901857fcd118f065f900982c24"
```
[the remark this answers is not in the document]
Comment (valijec) [reply recipes-multimedia/gstreamer/imx-gst1.0-plugin_git.bb]: > Please move this assignment under the `LICENSE` assignment
Thank you, done :)

**Comment by valijec** [comment]
> There are some other changes in meta-imx that we might need, like `pipewire`

Thank you @thochstein I will investigate and I'll open a new PR if it needs to be added

**Pull request merged**